src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/GetAzureSqlDatabaseAuditingPolicy.cs

```diff
@@ -21,6 +22,7 @@ namespace Microsoft.Azure.Commands.Sql.Auditing.Cmdlet
     /// Returns the auditing policy of a specific database.
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "AzureRmSqlDatabaseAuditingPolicy", SupportsShouldProcess = true), OutputType(typeof (AuditingPolicyModel))]
+    [Obsolete]
```

Comment:
Also, please add OBSOLETE, and a pointer to the new cmdlet: to the help synopsis / description